lua/nvim-treesitter/parsers.lua

```diff
@@ -1976,6 +1976,14 @@ list.typoscript = {
   maintainers = { "@Teddytrombone" },
 }
 
+list.typst = {
+  install_info = {
+    url = "https://github.com/uben0/tree-sitter-typst.git",
+    files = { "src/parser.c", "src/scanner.c" },
+  },
+  maintainers = { "@uben0" },
```

Comment:
"Maintainer" here means the person maintaining **the queries** for Neovim (reviewing issues/PRs about them, fixing them when the parser introduces breaking changes). That is usually not the person writing the grammar (and rarely appreciated to nominate someone else to do work!)

Reply:
Sorry, thought that refers to the maintainer of the parser.